src/metrics.rs

```diff
@@ -2308,6 +2399,9 @@ impl GooseAttack {
                 }
             }
 
+            // Record current users for users per second graph in HTML report.
+            self.metrics.record_users_per_second();
```

Comment:
I still think this should be moved down to _after_ we receive metrics, as otherwise we may not record a user until later than we should? It can be addressed in a follow, or let's talk through why this isn't relevant.